examples/using-redux/src/html.js

```diff
@@ -1,19 +1,11 @@
 import React, { Component } from "react"
```

Comment:
I think we can delete this whole file? It doesn't seem to be doing anything different than the default in  `.cache/default-html.js`.